assets/js/lib/operations/index.js

```diff
@@ -8,10 +8,11 @@ export const SAP_SYSTEM_OPERATION = 'sap_system';
 export const APPLICATION_INSTANCE_OPERATION = 'application_instance';
 export const CLUSTER_HOST_OPERATION = 'cluster_host';
 export const DATABASE_OPERATION = 'database';
-
```

Comment:
nitpick:
could we restore this new line?
To have a logical division between operation scopes and operations themselves